core/src/proof.rs

```diff
@@ -18,7 +18,10 @@ pub struct DataIntegrityProof {
 
 /// Given a JSON-LD document, create a data integrity proof for the document.
 /// Currently, only `Ed25519Signature2018` data integrity proofs in the JSON-LD format can be created.
-pub fn create_data_integrity_proof<S: signature::suite::Signature>(
+pub fn create_data_integrity_proof<
+    S: signature::suite::Signature,
+    // T: signature::keypair::Ed25519SSIKeyPair,
```

Comment:
Why is this commented out?

Reply:
It's a leftover from the previous implementation